### ISSUE TITLE
Fill in help for crux-jvm command-line options

### ISSUE THREAD

**Issue by atomb** [GaloisInc/crucible#448]
Some of the command-line options for `crux-jvm` have the help text `TODO`, which isn't very helpful.

**Comment by robdockins** [comment]
Indeed, the configuration for crux-jvm could use some love generally.  We should probably pay attention to the `CLASSPATH` env var, and all these options should be able to be specified from the configuration file.

**Comment by robdockins** [comment]
Actually, we should probably update the classpath loading code to be more Java-like, and find the standard library JAR using JAVA_HOME/JRE_HOME or similar instead of requiring it to be manually provided as part of the classpath.

`loadCodebase` has a 10-year old comment:
```
  -- REVISIT: Currently, classes found in the classpath shadow those in the
  -- jars.  Pretty sure the -classpath argument to the vanilla jvm allows
  -- mixture of jars and directories, and resolves names in the order in which
  -- those elements are encountered.  We probably want to do the same thing (and
  -- be able to just provide one argument to loadCodebase), but not for the
  -- beta. [js 04 Nov 2010]
```

**Comment by atomb** [comment]
I heartily endorse using more standard configuration techniques.

**Comment by RyanGlScott** [comment]
I filled in the TODOs in 950a629478354204f2cb72e46a4dc3078b5a459a, and `crux-jvm` now uses the version of Java on your `PATH` as of bd2ed44165ee4fedcf41ed53cb2d2fbae79796af. I suppose that only leaves the `JAVA_HOME`/`JRE_HOME` suggestion, but now that `crux-jvm` consults the `PATH`, I'm not sure if it's still needed.
